src/scripting/LuaSandbox.cpp

```diff
@@ -507,7 +507,8 @@ void LuaSandbox::InitializeIOForSandbox(Sandbox& aSandbox, const sol::state& acp
         for (const auto& entry : std::filesystem::directory_iterator(path))
         {
             sol::table item(stateView, sol::create);
-            item["name"] = UTF16ToUTF8(relative(entry.path(), path).native());
+            item["relativePath"] = UTF16ToUTF8(Relative(entry.path(), path).native());
```

Comment:
do we really need this if usecase is with dir? cant we somehow just do this in the dir command if it is required for it?

Reply:
I can remove it. It's not used anywhere